### ISSUE TITLE
[Feature Request] Show video progress bar when the UI is locked

### ISSUE THREAD

**Issue by eredrolo** [mpv-android/mpv-android#396]
When the UI is locked, only the unlock button is displayed. I would like if the video progress bar is also displayed.

**Comment by sfan5** [comment]
Show just the progress bar but not the rest of the OSC? This seems a bit weird, do any other players do this?

**Comment by eredrolo** [comment]
Yes. Show the progress bar along with the unlock button when I tap on the screen. Here's a screenshot from VLC.

![Screenshot_20210329-005548](https://user-images.githubusercontent.com/74611933/112765288-23e26d80-902a-11eb-8eea-9b27a6b2b080.jpg)
